src/main/webapp/script.js

```diff
@@ -39,21 +45,23 @@ function loadBody() {
 
 /** Contacts Login servlet to verify if user is logged in. */
 function verifyLogin() {
-  fetch('/login').then(response => response.json()).then((userDetails => {
-    const loginElement = document.getElementById("login-box");
-    const schoolName = document.getElementById("school-name");
-    if (!userDetails.loggedIn) {
-      schoolName.style.display = "none";
-      loginElement.innerHTML += `<p><a href= ${userDetails.loginURL} > Click here to log in! </a></p>`;
-    } else {
-      console.log(userDetails);
-      schoolName.style.display = "block";
-      if (userDetails.hasOwnProperty("schoolName")){
-        schoolName.innerHTML += userDetails.schoolName;
+  fetch("/login")
+    .then((response) => response.json())
+    .then((userDetails) => {
+      const loginElement = document.getElementById("login-box");
+      const schoolName = document.getElementById("school-name");
+      if (!userDetails.loggedIn) {
+        schoolName.style.display = "none";
+        loginElement.innerHTML += `<p><a href= ${userDetails.loginURL} > Click here to log in! </a></p>`;
+      } else {
+        console.log(userDetails);
+        schoolName.style.display = "block";
```

Comment:
instead of changing style to "block" or "none", it's better style to add or remove a class that hides the element. I have this in my css right now - I can show/explain to you on a call or you can look @ my code if that's helpful!

Reply:
Ohhh yes yes, this actually got merged from a previous function you had I think. Don't need it for my code so will delete now :)